_component_design/switches.md

```diff
@@ -72,18 +91,25 @@ Best practice form design does not allow a switch to exist in an error state, so
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
 {% figure [caption:"Detailed specifications for on and off states of a switch"] [class:"image bg-light border"] %}
 ![]({{site.url}}/assets/images/components/inputs-and-controls/switches/switch-states.svg)
 {% endfigure %}
+
 {% endcolumn %}
 
 </div>
 
-## Context
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+## Context
```

Comment:
Convert to H3

Reply:
I'm thinking this should stay as H2 here because the content doesn't really fall under the `States` header. Moving it above the column.